mmv1/third_party/terraform/tests/resource_composer_environment_test.go.erb

```diff
@@ -1011,16 +1011,17 @@ func testAccComposerEnvironment_nodeCfg(environment, network, subnetwork, servic
 	return fmt.Sprintf(`
 resource "google_composer_environment" "test" {
 	name   = "%s"
-	region = "us-central1"
+	region = "us-east1"  # later should be changed to us-central1, when feature is accessible globally
```

Comment:
Should we specify we are talking about the `enable_ip_masq_agent` feature specifically?